SYCL/Basic/partition_supported.cpp

```diff
@@ -6,12 +6,15 @@
 // Nvidia should not allow sub_devices but does not throw corresponding error.
 // XFAIL: hip_nvidia
 /* Check that:
-1) [info::device::partition_properties]: returns the partition properties
+1) if partition_equally is supported, then we check that the correct
+invalid errc is returned if more than max_compute_units are requested
+
+2) [info::device::partition_properties]: returns the partition properties
```

Comment:
Should this comment be more specific now since it only seems to apply to `info::partition_property::partition_by_affinity_domain`? Also, would it make sense to add some comments to the different test blocks cross-referencing them to this list?